components/common/layout/Footer.js

```diff
@@ -29,19 +26,19 @@ const Footer = () => (
     <div className={l('nested1')}>
       <div className={l('nested2')}>
         <div className={cn(f('links'), l('block1'))}>
-          <div className={f('links-column')}>
+          {/* <div className={f('links-column')}>
             <div className={f('header')}>
               <Text id="footer.materials" />
             </div>
-            {/* `themes` should be first */}
+            // `themes` should be first
             {TOPICS.map(topic => (
               <div key={topic} className={f('item')}>
                 <Link route={ROUTES_NAMES.topic} params={{ topic }}>
                   <Text id={`topic.by_${topic}`} />
                 </Link>
               </div>
             ))}
-          </div>
+          </div> */}
```

Comment:
?

Reply:
We are replacing this block with "podcasts platforms" block but I guess we might want to quickly switch it back again.